test/tests/functional/pbs_qrun.py

```diff
@@ -36,6 +36,7 @@
 # trademark licensing policies.
 
 from tests.functional import *
+import time
```

Comment:
You don't need to import time.  PTL does that for you.

Reply:
ok will address this in my next commit.